Sources_v3/Utils/Timers.swift

```diff
@@ -27,6 +27,15 @@ protocol Timer {
         queue: DispatchQueue,
         onFire: @escaping () -> Void
     ) -> RepeatingTimerControl
+    
+    /// Returns the current date and time.
+    static func currentTime() -> Date
+}
```

Comment:
I'm not sure it makes sense to add this to timer, they are not related, when you think about it. I think we should have a separate entity for this: `Time`.

Reply:
Actually I use your solution for tests in `v2`. It's good for comparison with the current date somewhere.

Reply:
"my" solution doesn't mean it's a good/correct solution.

Reply:
But I like it and it works 😄